Misc/NEWS.d/next/Security/2021-03-30-16-29-51.bpo-36384.sCAmLs.rst

```diff
@@ -0,0 +1,6 @@
+:mod:`ipaddress` module no longer accepts any leading zeros in IPv4 address
+strings. Leading zero are ambiguous and interpreted as octal notation by
+some libraries. The old function :func:`socket.inet_aton` treats leading
```

Comment:
```suggestion
some libraries. The function :func:`socket.inet_aton` treats leading
```

Reply:
I replaced ``old`` with ``legacy`` to make clear that ``inet_aton`` is the problematic function from the past and ``inet_pton`` is the new, better way.